pkg/cluster/config.go

```diff
@@ -32,6 +33,20 @@ const (
 	MachineAPINamespace = "openshift-machine-api"
 )
 
+var (
+	// proxyEnvVars is a list of the supported proxy-related environment variables
+	proxyEnvVars = []string{"HTTP_PROXY", "HTTPS_PROXY", "NO_PROXY"}
```

Comment:
Does this need to be a global variable?
Seems like its specific to the `getProxyVars` function

Reply:
Guess I was thinking ahead here. This var will likely be exported+used in future proxy PRs. I'll move it to getProxyVars()